### PR TITLE
few fixes

### PR DESCRIPTION
**Pull request by ovari**
* Hopefully each menu item as a keyboard shortcut (accelerator)
* https://github.com/jeromerobert/pdfarranger/issues/223

Thank you

### CODE REVIEW

File: po/hu.po [hunk 1]
```diff
@@ -160,55 +160,55 @@ msgstr "Kicsinyítés"
...
+msgstr "Kijelölés e_xportálása…"
 
 #: data/menu.ui.h:5
 msgid "_Undo"
 msgstr "_Visszavonás"
 
 #: data/menu.ui.h:6
 msgid "_Redo"
-msgstr "_Ismét"
+msgstr "Uj_ra"
```
[the remark this answers is not in the document]
Comment (dreua) [reply po/hu.po:191]: There should be no restriction of usable characters (the Japanese translation uses a lot of "special" characters and I haven't heard of problems there). Let me know if there should be any technical difficulties.

File: data/com.github.jeromerobert.pdfarranger.desktop [hunk 2]
```diff
@@ -1,15 +1,18 @@
 [Desktop Entry]
 Version=1.0
 Name=PDF Arranger
+Name[hu]=PDF-fájlok elrendezése (PDF Arranger)
```
[the remark this answers is not in the document]
Comment (ovari) [reply data/com.github.jeromerobert.pdfarranger.desktop:4]: Done, thanks.

File: po/hu.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (ovari) [reply po/hu.po]: Done, thanks.

File: data/com.github.jeromerobert.pdfarranger.desktop (hunk 2 above)
[the remark this answers is not in the document]
Comment (ovari) [reply data/com.github.jeromerobert.pdfarranger.desktop]: Done, thanks.

tl;dr
1. As this program can also work with images, our suggestion of "PDF-fajlok" is incorrect.
2. This program is not a full PDF editor "szerkesztő".
3. Thanks for the "PDF-rendező" suggestion which seems better and is also shorter.

**Comment by ovari** [comment]
@juhdanad if you are happy with the translation, can you please merge this now? Thank you

**Comment by juhdanad** [comment]
Yes, I am happy with the translation! I am not a maintainer, so I can't merge anything.

**Comment by jeromerobert** [comment]
981b664 b2088d0

**Comment by ovari** [comment]
Thanks for merging.

Can the lines from https://github.com/jeromerobert/pdfarranger/blob/master/po/hu.po#L277 onwards be deleted?

Thank you once again

**Comment by jeromerobert** [comment]
done

**Comment by ovari** [comment]
https://github.com/jeromerobert/pdfarranger/blob/master/po/pdfarranger.pot [1] has 318 lines.

https://github.com/jeromerobert/pdfarranger/blob/master/po/hu.po has 266 lines.
https://github.com/jeromerobert/pdfarranger/blob/master/po/hu.po#L44 `No document` isn't in [1]. Does this string need adding or should it be removed?

**Comment by jeromerobert** [comment]
I'll explain this in the README
